applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py

```diff
@@ -113,89 +113,81 @@ def _CreateSolver(self, external_model_part=None):
             ## Read the model - note that SetBufferSize is done here
             self.solver.ReadModelPart() # TODO move to global instance
 
-    def _InitializeIO(self):
-        """ Initialize GiD  I/O """
-        self.output_post  = self.ProjectParameters.Has("output_configuration")
-        if (self.output_post == True):
+    def _SetUpGiDOutput(self):
```

Comment:
What if we wanted to export to hdf5 ? the name does not suggest it

Reply:
I discussed this with @jcotela , this will be improved in a later PR

We have sth more general in mind that will be in the baseclass

Reply:
Then we will have a proper factory instead of this function (that is the plan for the fluid, at least...).